server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/QProfileTester.java

```diff
@@ -50,7 +50,7 @@
   }
 
   public QualityProfilesService service() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "QualityProfilesService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)